demand-dashboard/public/analysis.js

```diff
@@ -1,15 +1,66 @@
 $(function() {
-  // todo: use a default center based on data
-  var mapCenter = [-73.9440917, 40.7682802];
-  $("#latitude-input").val(mapCenter[1]);
-  $("#longitude-input").val(mapCenter[0]);
+    // todo: use a default center based on data
+    var mapCenter = [-73.9440917, 40.7682802];
+    $("#latitude-input").val(mapCenter[1]);
+    $("#longitude-input").val(mapCenter[0]);
+
+    $('#date-slider').change(function (e) {
+        $('#date-slider-value').html("6/" + e.target.value.toString() + "/2017");
+    });
+
+    $('#time-slider').change(function (e) {
+        console.log(e.target.value)
+        var input = parseInt(e.target.value);
+        $('#time-slider-value').html(prettyNumbers(Math.floor(input / 2)) + ":" + prettyNumbers((input % 2) * 30) + "-" +
+            (prettyNumbers(Math.floor((input + 1) / 2))) + ":" + prettyNumbers(((input + 1) % 2) * 30))
+    });
+
+    function prettyNumbers(number) {
+        var result = number.toString()
+        if (result.length == 1) {
+            return "0" + result
+        }
+        return result
+    }
+
+    var actualDemandMap = new mapboxgl.Map({
+        container: 'actual-demand-map',
+        style: 'mapbox://styles/mapbox/streets-v9',
+        zoom: 11,
+        center: mapCenter
+    });
+
+    actualDemandMap.on('load', function () {
+        actualDemandMap.addSource("actualDemand", {
+            type: "geojson",
+            data: "https://s3.amazonaws.com/koober-5152/koober-training.json"
```

Comment:
I think it might be better to serve this static file from the dashboard app (to reduce external deps).  So copy it into the `public` folder.  Then you should be able to change this url to a relative one (using the Static Asset reverse router).